src/osd/OSDMap.cc

```diff
@@ -470,6 +470,9 @@ void OSDMap::Incremental::encode(bufferlist& bl, uint64_t features) const
     if (target_v >= 4) {
       ::encode(new_require_min_compat_client, bl);
     }
+    if (target_v >= 5) {
+      ::encode(new_require_osd_release, bl);
```

Comment:
shall we translate `new_require_osd_release` to the `new_flags` bits if `target_v < 5`?